modules/calc.js

```diff
@@ -0,0 +1,30 @@
+const { MessageType } = require("@adiwajshing/baileys");
+const Strings = require("../lib/db");
+const format = require("python-format-js");
+const inputSanitization = require("../sidekick/input-sanitization");
+const calc = Strings.calc;
+const math = require("mathjs");
+
+module.exports = {
+    name: "calc",
+    description: calc.DESCRIPTION,
+    extendedDescription: calc.EXTENDED_DESCRIPTION,
+    demo: { isEnabled: true, text: ".calc 2^4+5!" },
+    async handle(client, chat, BotsApp, args) {
+        let output = math.evaluate(args[0]);
+        if(!isNaN(output)) {
```

Comment:
Having `!` negates the condition, which equates to if not *is not a number*, then say invalid syntax. This means that if the output is a number, then say invalid syntax, which is not what we want.
```suggestion
        if(isNaN(output)) {
```